src/ocean/util/serialize/contiguous/Serializer.d

```diff
@@ -622,6 +622,7 @@ struct Serializer
                 {
                     // element is a dynamic array
                     data = This.dumpArray(element, data);
+                    *(cast(U*)&element) = null;
```

Comment:
Out of curiosity - is there any point in doing so other than simplifying debugging? Deserializer doesn't care if pointers are null or garbage.